tailwind.config.js

```diff
@@ -63,6 +64,20 @@ module.exports = {
               height: 'auto',
               aspectRatio: '16/9',
             },
+
+            figure: {
+              textAlign: 'center',
+            },
+
+            figcaption: {
+              lineBreak: 'anywhere',
+              wordBreak: 'normal',
+              overflow: 'hidden',
+              whiteSpace: 'nowrap',
+              textOverflow: 'ellipsis',
```

Comment:
Why do we use an ellipsis to hide the `figcaption` partially and is there a way to see it completely?

Reply:
If we don't want the size of the container to cut the paragraph, we can remove the `whiteSpace` property. Otherwise it's truncated and `ellipsis` property will add "...". Perhaps the better option would be, as you suggested, to have it fully readable across all screen devices.